applications/nublado/templates/tutorials-cronjobs.yaml

```diff
@@ -104,11 +111,19 @@ spec:
             - name: "GIT_BRANCH"
               value: {{ .Values.cronjob.artifacts.gitBranch | quote }}
             volumeMounts:
-            - mountPath: {{ .Values.cronjob.artifacts.targetVolume.mountPath }}
-              name: target
+            - mountPath: {{ .Values.cronjob.artifacts.targetVolume.mountPath | quote }}
+              name: {{ .Values.cronjob.tutorials.targetVolume.volumeName | quote }}
```

Comment:
Should this be:

```suggestion
              name: {{ .Values.cronjob.artifacts.targetVolume.volumeName | quote }}
```

Reply:
Only relevant if the artifacts will also change to use a `volumeName`
I think this is a bug, but was the intention for tutorials and artifacts to share the same volume?

Reply:
Oh.  They probably always will, but the config shouldn't assume that.